reference/import_export/simple-import.rst

```diff
@@ -0,0 +1,42 @@
+How import works
+================
+
+.. _Icecat: https://github.com/akeneo/pim-community-dev/tree/master/src/Akeneo/Bundle/BatchBundle
```

Comment:
Icecat which links to BatchBundle? You want link to icecat_demo_dev in pim community dev no? btw when installing a pim standard we install the minimal set, means very required entities to make the pim works.

btw you can re-work a bit the https://github.com/akeneo/pim-docs/blob/master/cookbook/setup_data/customize_installer.rst even the yml example we give is deprecated